locations/hoenn/encounters.json

```diff
@@ -4025,6 +4120,31 @@
           }
         ]
       },
+      {
+        "name": "Seafloor Cavern 8",
+        "access_rules": ["$route_124_access,$dive,sea_floor_grunts_on","$route_124_access,$dive,steven_gives_dive"],
```

Comment:
This should be seafloor_cavern_access to account for needing strength and rock smash